### PR TITLE
Milestone #4: Feature desktop UI version

### PR DESCRIPTION
**Pull request by fezzopro**
## Description
This PR contains instructions from [Portfolio: desktop version](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md)

## introduced changes

* Linter configuration files from [Microverse Linter](https://github.com/microverseinc/linters-config)  
* [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
* Portfolio Desktop changes reflected on:
  * index.html
  * style.css

## Project requirements

- Use media queries in your CSS file to make your desktop version look like your selected template (Desktop: >=768px).
- To lay out the elements on the page you should use Grid and Flexbox.
- You don't need to implement any functionality that requires JavaScript.
- You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -370,3 +393,170 @@ a:hover {
...
+  }
+
+  .headline h1,
+  .headline h2 {
+    padding-top: 7.125rem;
+  }
+}
+
+/* Desktop version */
+@media only screen and (min-width: 768px) {
```
Comment (braincee) [inline style.css:413]: - [ ] Please I recommend sticking to the [FIGMA DESIGN TEMPLATE]. 😃 

_**REQUIRED FIGMA DESIGN**_

Screenshot (162)


_**YOUR DESIGN**_

- [ ] Please I recommend implementing changes on your media queries `min-width: 768px`  for your `form` section.
- [ ] Please I recommend changing the button text from `Get my resume` to `Get in touch`.

Screenshot (163)
.

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @fezzopro 👋 , I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (fezzopro) [reply style.css:413]: Thanks for your feedback 👍

**Review by algerina — APPROVED** [review]
Hi @fezzopro  ,

Great job so far! 

May I highlight the following: 
- [x] The project is set correctly✔️ 
- [x] Well done implementing the previous reviewer's requirements✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**